<ui version="4.0" >
 <class>JoinMultiChatDialogClass</class>
 <widget class="QDialog" name="JoinMultiChatDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Join groupchat</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>3</number>
   </property>
   <property name="topMargin" >
    <number>3</number>
   </property>
   <property name="rightMargin" >
    <number>3</number>
   </property>
   <property name="bottomMargin" >
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblIdentity" >
       <property name="text" >
        <string>Identity:</string>
       </property>
       <property name="buddy" >
        <cstring>cmbStreamJid</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbStreamJid" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblResent" >
       <property name="text" >
        <string>Recent:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbRecent" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tlbRecentDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grbRoom" >
     <property name="title" >
      <string>Room</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lblHost" >
        <property name="text" >
         <string>Host:</string>
        </property>
        <property name="buddy" >
         <cstring>lneHost</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QLineEdit" name="lneHost" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblRoom" >
        <property name="text" >
         <string>Room:</string>
        </property>
        <property name="buddy" >
         <cstring>lneRoom</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QLineEdit" name="lneRoom" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lblNick" >
        <property name="text" >
         <string>Nick:</string>
        </property>
        <property name="buddy" >
         <cstring>lneNick</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="lneNick" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="lblPassword" >
        <property name="text" >
         <string>Password:</string>
        </property>
        <property name="buddy" >
         <cstring>lnePassword</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2" >
       <widget class="QLineEdit" name="lnePassword" >
        <property name="echoMode" >
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QToolButton" name="tlbResolveNick" >
        <property name="text" >
         <string>Resolve</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="lineHorizontal" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btbButtons" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>cmbStreamJid</tabstop>
  <tabstop>cmbRecent</tabstop>
  <tabstop>lneHost</tabstop>
  <tabstop>lneRoom</tabstop>
  <tabstop>lneNick</tabstop>
  <tabstop>lnePassword</tabstop>
  <tabstop>btbButtons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>JoinMultiChatDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>356</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel" >
     <x>380</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
